test/end2end_test.go

```diff
@@ -3926,8 +3926,37 @@ func (s) TestClientInvalidStreamID(t *testing.T) {
 	st.greet()
 	st.writeHeadersGRPC(2, "/grpc.testing.TestService/StreamingInputCall", true)
 	goAwayFrame := st.wantGoAway(http2.ErrCodeProtocol)
-	want := "received an illegal stream id: 2. headers frame: [FrameHeader HEADERS flags=END_STREAM|END_HEADERS stream=2 len=60]"
-	if got := string(goAwayFrame.DebugData()); got != want {
+	want := "received an illegal stream id: 2."
+	if got := string(goAwayFrame.DebugData()); !strings.Contains(got, want) {
+		t.Fatalf("Error received: %v, want: %v", got, want)
```

Comment:
Nit: "`want: Contains(%v)`" or something to indicate what's happening.

Reply:
Done

Reply:
I see it changed below but not here.

Reply:
ah right. changed in the previous test as well. thanks